templates/shared_services/databricks-auth/template_schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/microsoft/AzureTRE/templates/shared_services/databricks-auth/template_schema.json",
+  "type": "object",
+  "title": "Azure Databricks Private Authentication Shared Service",
+  "description": "Azure Databricks Private Authentication Shared Service",
+  "required": [],
+  "properties": {
+    "display_name": {
+      "type": "string",
+      "title": "Name for the workspace service",
+      "description": "Azure Databricks Private Authentication Shared Service",
+      "default": "Azure Databricks Private Authentication Shared Service",
+      "updateable": true
+    },
+    "description": {
+      "type": "string",
+      "title": "Description of the workspace service",
+      "description": "Azure Databricks Private Authentication Shared Service",
+      "default": "Azure Databricks is a fast, easy, and collaborative Apache Spark-based big data analytics service designed for data science and data engineering.",
```

Comment:
Description should relate to what this enables.

Reply:
I've updated it to match AML bundle, but as a shared service, and provided default value which is related,
let me know if that's what you meant.